desk/app/activity.hoon

```diff
@@ -866,63 +726,22 @@
     ?~  last  `key.event
     last
   $(stream rest)
-++  get-order
-  |=  =source:a
-  %.  -.source
-  %~  got  by
-  ^~
-  %-  my
-  :~  [%thread 6]
-      [%dm-thread 5]
-      [%channel 4]
-      [%group 3]
-      [%dm 2]
-      [%base 1]
-  ==
-++  convert-to
-  |%
-  ++  activity-0
-    |=  =activity:a
-    ^-  activity-0:old:a
-    %-  ~(run by activity)
-    activity-summary-0
-  ++  activity-summary-0
-    |=  as=activity-summary:a
-    ^-  activity-summary-0:old:a
-    :*  newest.as
-        count.as
-        notify.as
-        unread.as
-        ?~  children.as  ~
-        `(activity-0 u.children.as)
-    ==
-  ++  update-0
-    |=  =update:a
-    ^-  update-0:old:a
-    ?+  -.update  update
-        %read
-      [%read source.update (activity-summary-0 activity-summary.update)]
-    ==
-  --
+::
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::  repent ye, for the sins of the past have come to haunt us         ::
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
```

Comment:
Such comment blocks are technically illegal, you know. (;

Most of the arms below already have their own paragraph(s) for each specific sin. If we really want a separator, may be more appropriate to use `+|` chapters.

Reply:
```suggestion
::
```